besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -832,6 +832,14 @@ void setBannedNodeIds(final List<String> values) {
       arity = "1")
   private final Wei txFeeCap = DEFAULT_RPC_TX_FEE_CAP;
 
+  @Option(
+      names = {"--require-tx-replay-protection"},
+      description =
+          "Requires transactions to be EIP-155 to be submitted via RPC (default: ${DEFAULT-VALUE})",
+      arity = "1")
```

Comment:
A few more nits here:
* Suggest moving this to `RPCOptions`
* Suggest renaming to `--strict-tx-replay-protection-enabled` to be more in line with our other boolean option names.  
* Would be nice to allow no params with a fall-back to true
* Suggest some tweaks to the description as well, see suggestion below

```suggestion
  @Option(
      names = {"--strict-tx-replay-protection-enabled"},
      description =
          "Require transactions submitted via JSON-RPC to use replay protection in accordance with EIP-155 (default: ${DEFAULT-VALUE})",
      fallbackValue = "true",
      arity = "0..1")
```

Reply:
Sounds good!